docs/source/reference/builtin-devices.rst

```diff
@@ -40,22 +40,38 @@ This must be sub-classed (like :class:`~ophyd.device.Device`) to be useful.
    ophyd.epics_motor.EpicsMotor
    ophyd.epics_motor.MotorBundle
 
+.. _built_in.Epics.Scaler.record:
+
+EPICS Scaler
+------------
+
+The EPICS ``scaler`` record ([#scaler]_) is supported by two alternative
+``ophyd`` Devices.  (You need only choose one of these two.)  An important
+difference between the :ref:`built_in.EpicsScaler` and the :ref:`built_in.ScalerCH`
+Devices is in how each channel's names is represented, as summarized 
```

Comment:
"names are represented" or "name is represented"?

Reply:
good catch